.github/workflows/check-manual.yml

```diff
@@ -54,9 +54,4 @@ jobs:
       run: |
         cd manual/
         sed "s/REQUIREMENTS/$(sed -e 's/[\&/]/\\&/g' -e 's/$/\\n/' ../requirements.txt | tr -d '\n')/" index-rst-template > index.rst
-        sphinx-build -b html . build 1>out.txt 2>err.txt
-        echo "stdout:"
```

Comment:
I have edited this so that the error message is given as an output instead of being saved to a file.

Was there a reason to save this information to a file? Let me know if its okay to remove these lines.

Reply:
Yes, it was because I wanted to fail for warnings, which would go to stderr but wouldn't cause the command to fail.
I think it would be better to print stderr _and_ have it go to a file. See for example [here](https://stackoverflow.com/questions/26915448/how-do-i-copy-stderr-without-stopping-it-writing-to-the-terminal) for how to do this.

Reply:
I managed to get the build to fail for warnings by doing

`sphinx-build -W -b html . build`

The -W flag ensures warnings are treated as errors. I artificially introduced a warning and then tried this on the CI and the build does indeed fail due to the warning being treated as an error.

See the CI run https://github.com/CQCL/pytket/actions/runs/3376800393/jobs/5604961231

Is this acceptable behavior? If preferred I can still make it print out the `err.txt` file

Reply:
Yes that sounds like a good solution, thanks.